src/components/Security/Users/Steps/UserProfileList.vue

```diff
@@ -48,29 +47,29 @@
 </template>
 
 <script type="text/javascript">
-import MSelect from '../../../Common/MSelect'
 import { performSearchProfiles } from '../../../../services/kuzzleWrapper'
 
 export default {
   name: 'UserProfileList',
-  components: {
-    MSelect
-  },
+  components: {},
   props: {
     addedProfiles: {
       type: Array
     }
   },
   data() {
     return {
-      profileList: []
+      profileList: [],
+      selectedProfiled: 0
     }
   },
   computed: {
     availableProfiles() {
-      return this.profileList.filter(profile => {
-        return this.addedProfiles.indexOf(profile.id) === -1
-      })
+      return this.profileList
+        .filter(profile => {
+          return this.addedProfiles.indexOf(profile.id) === -1
```

Comment:
Nitpicking: prefer `Array.includes`

```suggestion
          return !this.addedProfiles.includes(profile.id)
```